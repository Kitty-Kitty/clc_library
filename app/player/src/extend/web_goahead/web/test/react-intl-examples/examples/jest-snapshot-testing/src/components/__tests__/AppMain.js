/*global expect*/

import React from 'react';
import createComponentWithIntl from '../../utils/createComponentWithIntl';
import AppMain from '../AppMain';

test('app main should be rendered', () => {
    const component = createComponentWithIntl(<AppMain/>);

    let tree = component.toJSON();

    expect(tree).toMatchSnapshot();

    tree.props.onClick();

    tree = component.toJSON();

    expect(tree).toMatchSnapshot();
});
