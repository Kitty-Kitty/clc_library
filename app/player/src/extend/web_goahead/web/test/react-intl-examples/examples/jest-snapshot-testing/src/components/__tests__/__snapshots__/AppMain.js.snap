exports[`test app main should be rendered 1`] = `
<main
  className="app-main"
  onClick={[Function]}>
  <div
    className="app-main__counter">
    <span>
      Counter 1
    </span>
  </div>
</main>
`;

exports[`test app main should be rendered 2`] = `
<main
  className="app-main"
  onClick={[Function]}>
  <div
    className="app-main__counter">
    <span>
      Counter 2
    </span>
  </div>
</main>
`;
